lib/connection.h

```diff
@@ -700,11 +700,14 @@ public Q_SLOTS:
     virtual LeaveRoomJob* leaveRoom(Room* room);
 
 #ifdef Quotient_E2EE_ENABLED
-    void startKeyVerificationSession(const QString& deviceId);
+    void startKeyVerificationSession(const QString& userId, const QString& deviceId);
 
     void encryptionUpdate(Room *room);
 #endif
 
+    // WARNING: Use only for unit tests! This will not lead to a correctly working connection!
+    void completeSetup(const QString& mxId);
```

Comment:
...and then you don't need that warning :)
```suggestion
     static Connection* makeMockConnection(const QString& mxId);
```